tools/ci/tc/decision.py

```diff
@@ -74,6 +74,17 @@ def get_run_jobs(event):
     path_jobs = jobs.get_jobs(paths)
     all_jobs = path_jobs | get_extra_jobs(event)
     logger.info("Including jobs:\n * %s" % "\n * ".join(all_jobs))
+
+    # Exclude browser stability tests for exports from that specific browser.
+    try:
```

Comment:
Can we put this in a `filter_jobs` function, and call it before the `Including jobs` log line?

Reply:
Great idea - implemented